script/package-worker.ps1

```diff
@@ -14,14 +14,13 @@ foreach ($framework in $frameworks)
         Copy-Item "$($runtime.FullName)\*" -Destination $worker_version_dir -Recurse
         $filename = "Microsoft.Spark.Worker.$framework.$runtime-$version"
 
+        # Generate additional tar.gz worker files only for linux-x64.
```

Comment:
Given zip is universally available, do we lose anything by going with zip instead of tar.gz everywhere?